tests/functional/error-managed-identity/expected-stderr

```diff
@@ -0,0 +1 @@
+Query execution error: %AManaged Service Identity has not been enabled on this server. Please enable Managed Service Identity and try again.%A
```

Comment:
Nemame to v nasej CI Synapse zapnute, ale mozeme aspon testovat, ze to neprejde.